hax/hax/util.py

```diff
@@ -496,13 +503,15 @@ def get_svc_status(self, srv_fid: Fid, kv_cache=None) -> str:
 
     @supports_consul_cache
     def get_m0d_statuses(self,
+                         m0d_services=None,
                          kv_cache=None
                          ) -> List[Tuple[ServiceData, ServiceHealth]]:
         """
         Return the list of all Motr service statuses according to Consul
-        watchers. The following services are considered: ios, confd.
+        watchers.The following services are considered [default]: ios, confd.
         """
-        m0d_services = set(['ios', 'confd'])
+        if not m0d_services:
```

Comment:
let's rename `m0d_services` to `motr_services`.

Reply:
done